everviz/plugins/_crossplot/_callback/_crossplot_callback.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import plotly.graph_objs as go
+
+
+def _crossplot_update_graph(
```

Comment:
This is marked as a protected member of the module (the `_` in front of the name) but it is used outside of the module so is there a reason why it needs to be marked as protected?